README.md

```diff
@@ -1,7 +1,33 @@
+# OpenShift Serverless Previews Documentation
 
-# Knative Eventing on OpenShift Container Platform <img src="https://github.com/openshift-knative/docs/blob/master/images/knative-openshift-logo.png" width="100" height="50" />
+OpenShift Serverless Previews provides tutorials and information about
+additional, advanced use cases and integrations that are possible using
+OpenShift Container Platform and OpenShift Serverless, but are for development
+purposes only and are not officially supported by Red Hat.  This repository
+contains the source files for the OpenShift Serverless Previews documentation
+found at https://openshift-knative.github.io.
```

Comment:
This link doesn't work, needs to be https://openshift-knative.github.io/docs I think